osier/equations.py

```diff
@@ -0,0 +1,160 @@
+        The list of technologies.
+    dispatch_results : :class:`pandas.DataFrame`
+        The results dispatch results for each technology in 
+        :attr:`technology_list` from a :class:`osier.DispatchModel` run.
+    """
+
+    dispatchable_techs = [t for t in technology_list if t.dispatchable]
+
+    return dispatchable_techs
+
+
+def get_nondispatchable_techs(technology_list):
+    """
+    Parameters
+    ----------
+    technology_list : list of :class:`osier.Technology` objects
+        The list of technologies.
+    dispatch_results : :class:`pandas.DataFrame`
+        The results dispatch results for each technology in 
+        :attr:`technology_list` from a :class:`osier.DispatchModel` run.
+    """
+
+    dispatchable_techs = [t for t in technology_list if not t.dispatchable]
+
+    return dispatchable_techs    
+
+
+def get_dispatchable_names(technology_list):
+    """
+    Parameters
+    ----------
+    technology_list : list of :class:`osier.Technology` objects
+        The list of technologies.
+    dispatch_results : :class:`pandas.DataFrame`
+        The results dispatch results for each technology in 
+        :attr:`technology_list` from a :class:`osier.DispatchModel` run.
+    """
+
+    dispatchable_techs = [t.technology_name for t in technology_list if t.dispatchable]
+
+    return dispatchable_techs
+
+
+def annualized_capital_cost(technology_list, dispatch_model=None):
+    """
+    Parameters
+    ----------
+    technology_list : list of :class:`osier.Technology` objects
+        The list of technologies.
+    dispatch_model : :class:`osier.DispatchModel`
+        A _solved_ dispatch model (i.e. with model results and objective
+        attributes).
+    """
+    capital_cost = np.array([t.total_capital_cost / t.lifetime 
+                            for t in technology_list]).sum()
+
+    return capital_cost
+
+
+def annualized_fixed_cost(technology_list, dispatch_model=None):
+    """
+    Parameters
+    ----------
+    technology_list : list of :class:`osier.Technology` objects
+        The list of technologies.
+    dispatch_model : :class:`osier.DispatchModel`
+        A _solved_ dispatch model (i.e. with model results and objective
+        attributes).
+    """
+    fixed_cost = np.array([t.annual_fixed_cost
+                            for t in technology_list]).sum()
+
+    return fixed_cost
+
+
+def annual_co2(technology_list, dispatch_model, emission='co2'):
+    """
+    Parameters
+    ----------
+    technology_list : list of :class:`osier.Technology` objects
+        The list of technologies.
+    dispatch_model : :class:`osier.DispatchModel`
+        A _solved_ dispatch model (i.e. with model results and objective
+        attributes).
+    """
+    
+    dispatch_techs = get_dispatchable_techs(technology_list)
+    non_dispatch_techs = get_nondispatchable_techs(technology_list)
+    column_names = get_tech_names(technology_list)
+    dispatch_results = dispatch_model.results
```

Comment:
```suggestion
def annual_co2(technology_list, solved_dispatch_model, emission='co2'):
    """
    Parameters
    ----------
    technology_list : list of :class:`osier.Technology` objects
        The list of technologies.
    solved_dispatch_model : :class:`osier.DispatchModel`
        A _solved_ dispatch model (i.e. with model results and objective
        attributes).
    """
    
    dispatch_techs = get_dispatchable_techs(technology_list)
    non_dispatch_techs = get_nondispatchable_techs(technology_list)
    column_names = get_tech_names(technology_list)
    dispatch_results = solved_dispatch_model.results
```

Reply:
Okay, I'll update the other functions to reflect this as well.